db/seeds.rb

```diff
@@ -59,101 +59,18 @@ def seed_nonprofit_applications
-                            email: 'quintonddang@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Max', last_name: 'Wolffe',
-                            email: 'max.alan.wolffe@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Kevin', last_name: 'Shieh',
-                            email: 'kevshieh@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Warren', last_name: 'Shen',
-                            email: 'warrenzshen@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Will', last_name: 'Tang',
-                            email: 'williamctang93@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Aleks', last_name: 'Kamko',
-                            email: 'aykamko@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Varun', last_name: 'Rau',
-                            email: 'varun@berkeley.edu',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Nishant', last_name: 'Desai',
-                            email: 'nish.sdesai@gmail.com',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Jordeen', last_name: 'Chang',
-                            email: 'jordeenchang@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Sameera', last_name: 'Vemulapalli',
-                            email: 'sameera.vemulapalli@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Ethan', last_name: 'Chiou',
-                            email: 'nahte.uoihc@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Atsunori', last_name: 'Kakitani',
-                            email: 'atsunori.kakitani@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Elizabeth', last_name: 'Lin',
-                            email: 'elizabethylin@gmail.com',
-                            member_role: alumnus_role
-  Member.find_or_create_by! first_name: 'Andy', last_name: 'Qin',
-                            email: 'andrewlqin@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Jason', last_name: 'Won',
-                            email: 'jason.j.won@gmail.com',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Daniel', last_name: 'Li',
-                            email: 'danielmxli@gmail.com',
-                            member_role: developer_role
+  roles_hash = {}
```

Comment:
You forgot to include the first loop to create the `MemberRole`s, so `rake db:seed` is breaking right now (in fact, if you didn't transfer over the first / last name it would have never worked since we have validations on those).

Good question! In the earlier code, you wrote something like `Member.create! email: v["email"]`. To transfer over the name, you'd want to instead do something like

```ruby
Member.create! email: v["email"],
               first_name: v["first_name"],
               last_name: v["last_name"]
```

The reason why my code works is because `member_data` is just that hash. You can double check by adding `puts` statements and reseeding your DB. I use `#merge` so that we set the `member_role` correctly as well.